### PR TITLE
Crep fix/build

### PR DESCRIPTION
**Pull request by crepella**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+# Install git
+RUN apk update && apk add --no-cache git
+
+# Copy the entire project into the container
+COPY . .
+
+# Install app dependencies
+RUN npm install
+RUN npm install @vue/cli@3.7.0 -g
```
Comment (chimera-defi) [inline Dockerfile:18]: npm i should install all deps.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+RUN node -v
+RUN npm -v
```
Comment (chimera-defi) [inline Dockerfile:21]: y tho? just debug? rm plz

File: babel.config.js [hunk 3]
```diff
@@ -1,5 +1,9 @@
 module.exports = {
   presets: [
-    '@vue/cli-plugin-babel/preset'
-  ]
+    '@vue/cli-plugin-babel/preset',
+    '@babel/preset-env',
```
Comment (chimera-defi) [inline babel.config.js:4]: is this a no op change?

File: test-aliases.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+// test-aliases.js
+console.log(require.resolve('bnc-onboard'));
```
Comment (chimera-defi) [inline test-aliases.js:2]: toss it

File: test-import.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+// test-import.js
+const Onboard = require('bnc-onboard');
```
Comment (chimera-defi) [inline test-import.js:2]: toss

File: vue.config.js [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+        '@metamask/abi-utils',
+        'superstruct',
+        '@ethereumjs/util'
+      ],
+      configureWebpack: {
+        resolve: {
+            alias: {
+                'bnc-onboard': path.resolve(__dirname, 'node_modules/bnc-onboard/dist/bnc-onboard.min.js'),
+                '@ethereumjs/util': path.resolve(__dirname, 'node_modules/@ethereumjs/util/dist/index.browser.js'),
+                '@metamask/abi-utils': path.resolve(__dirname, 'node_modules/@metamask/abi-utils/lib/index.js')
```
Comment (chimera-defi) [inline vue.config.js:31]: did this get rid of the loader issues?

**Review by chimera-defi — CHANGES_REQUESTED** [review]

**Comment by chimera-defi** [comment]
can we add how to run the dockerfile / how youre doing it to the readme? or as a `docker` cmd in package.json plz?

**Comment by crepella** [comment]
Commands I used are 

docker build -t sharedstake-ui .
docker run -p 8080:8080 sharedstake-ui

**Comment by crepella** [comment]
Added the instructions to README under 'Run with Docker'

**Pull request closed without merging**